avocado_vt/plugins/vt_joblock.py

```diff
@@ -105,6 +105,11 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
+            files = list(self._get_lock_files())
```

Comment:
Btw the reason that it's rare to occur also means, users will forget about what the failure means and how to get details. So I'd really prefer providing as much info as we can (based on my experience).